README.md

```diff
@@ -199,7 +199,7 @@ APIs marked with the `@Deprecated` annotation at the class or method level will
 
 #### io.reactivex.internal.*
 
-All code inside the `io.reactivex.internal.*` packages is considered private API and should not be relied upon at all. It can change at any time. 
+All code inside the `io.reactivex.internal.*` packages are considered private API and should not be relied upon at all. It can change at any time. 
```

Comment:
The grammar parse is (approximately using no specific notation, excluding the second phrase):

> [[All code [inside the `io.reactivex.internal.*` packages]] is [considered private API]]

_[inside the `io.reactivex.internal.*` packages]_ is a qualifier describing the location of _All code_. When we remove it we are left with:

> [[All code] is [considered private API]]

which is correct as _code_ is a [mass noun](https://en.wikipedia.org/wiki/Mass_noun) in English.